packages/component-base/index.d.ts

```diff
@@ -1,11 +1,6 @@
-export { ActiveMixin } from './src/active-mixin.js';
 export { ControllerMixin } from './src/controller-mixin.js';
+export { DelegateStateMixin } from './src/delegate-state-mixin.js';
```

Comment:
Missing dependency in `package.json`.

Reply:
This mixin wasn't moved, it is still placed in `@vaadin/component-base`.